retry_test.go

```diff
@@ -82,7 +82,7 @@ func TestAttemptWithDelay(t *testing.T) {
 	})
 
 	is.Equal(iter1, 1)
-	is.Greater(dur1, 0*time.Millisecond)
+	is.GreaterOrEqual(dur1, 0*time.Millisecond)
```

Comment:
How could be invalid?

I mean this test will always pass. A duration is always positive, no?

Reply:
Could be my hardware or the fact that I'm running on Windows, but for me the first call to `AttemptWithDelay` (which succeeds immediately) returns within the minimum monotonic time period, so these tests always fail.